equinix/resource_metal_device_acc_test.go

```diff
@@ -747,30 +749,6 @@ resource "equinix_metal_device" "test" {
 `, confAccMetalDevice_base(preferable_plans, preferable_metros, preferable_os), projSuffix, rInt, rInt, rInt, testDeviceTerminationTime())
 }
 
-func testAccMetalDeviceConfig_varname_pxe(rInt int, projSuffix string) string {
```

Comment:
Was this test deleted by accident or does it need to be removed for some reason?

Reply:
Yes, this test helper was not being used anywhere. Should have called that out